galaxyui/src/app/content-detail/collection-detail/collection-detail.component.html

```diff
@@ -1 +1,153 @@
+                        <ng-template #headerTemplate>
+                            <div class="card-heading">
+                                <div>
+                                    <div class="icon-container">
+                                        <span class="fa fa-check-circle fa-2x"></span>
+                                    </div>
+                                    <div class="title-text">Community Score Details</div>
+                                </div>
+                                <div>
+                                    <span (click)="toggleSurveyDetails('showComunityDetails')" class="pficon-close fa-lg close-details"></span>
+                                </div>
+                            </div>
+                        </ng-template>
+
+                        <div class="card-body">
+                            <div *ngFor="let bar of communityScoreDetails">
+                                <app-score-bar
+                                    extraInfo="{{bar.count}} answer{{ bar.count !== 1 ? 's':'' }}"
+                                    [score]="bar.value"
+                                    [dividend]="5"
+                                    [text]="bar.question"
+                                    [customTextStyle]="{'width': '175px'}"
+                                    >
+                                </app-score-bar>
+                            </div>
+                        </div>
+                    </pfng-card>
+                </div>
+            </div>
+
+            <div *ngIf="showQualityDetails" class="row detail score-detail">
+                <div class="col-sm-12">
+                    <pfng-card
+                        [config]="config"
+                        [headerTemplate]="headerTemplate">
+                        <ng-template #headerTemplate>
+                            <div class="card-heading">
+                                <div>
+                                    <div class="icon-container">
+                                        <span class="fa fa-check-circle fa-2x"></span>
+                                    </div>
+                                    <div class="title-text">Quality Score Details</div>
+                                </div>
+                                <div>
+                                    <span (click)="toggleSurveyDetails('showQualityDetails')" class="pficon-close fa-lg close-details"></span>
+                                </div>
+                            </div>
+                        </ng-template>
+
+                        <div class="card-body">
+                            <table>
+                                <tr>
+                                    <th>Name </th>
+                                    <th>Quality Score </th>
+                                    <th>Content Score </th>
+                                    <th>Metadata Score </th>
+                                    <th>Compatibility Score </th>
```

Comment:
May want to not display Compatibility Score, as we don't display for traditional roles, calculation of this score is on hold (good to see how it is displayed in the mockups though)